src/nrc/datamodel/models.py

```diff
@@ -179,3 +180,80 @@ class NotificatieResponse(models.Model):
 
     def __str__(self) -> str:
         return f"{self.abonnement} {self.response_status or self.exception}"
+
+
+class CloudEvent(models.Model):
+    int_id = models.AutoField(
+        primary_key=True, serialize=False, verbose_name="ID", help_text=_("internal id")
+    )
+
+    id = models.CharField(_("id"), max_length=255, help_text=_("event id"))
+    source = models.CharField(_("source"), max_length=255, help_text=_("event source"))
+    specversion = models.CharField(
+        _("specversion"),
+        max_length=255,
```

Comment:
Currently this only allows major and minor (no patch versions), so I guess we can make the max_length a bit smaller here. It's stated that `Note: for 'release candidate' releases a suffix might be used for testing purposes.`, so we don't have to make `max_length=3`, but let's do something like 50 for now